src/screens/PageHeader.js

```diff
@@ -93,6 +93,19 @@ const PageHeaderPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="size">
+          <Description>
+            The possible sizes of PageHeader, which impacts the padding, title
+            sizing, and subtitle sizing. 'size' will not impact parent or action
+            sizing.
+          </Description>
+          <PropertyValue type="string">
+            <Example>small</Example>
+            <Example>medium</Example>
+            <Example>large</Example>
```

Comment:
```suggestion
            <Example>"small"</Example>
            <Example>"medium"</Example>
            <Example>"large"</Example>
```
This just matches how we have string examples other places in the docs